plugin/rename.py

```diff
@@ -27,12 +27,7 @@ def validate(self, name):
         return len(name) > 0
 
     def get_current_symbol_name(self):
-        pos = get_position(self.view)
-        current_name = self.view.substr(self.view.word(pos))
-        # Is this check necessary?
-        if not current_name:
-            current_name = ""
-        return current_name
+        return self.view.substr(self.view.word(get_position(self.view)))
```

Comment:
Have you checked wether `view.substr` always returns a `str`?

Reply:
Yes, I tested it in different edge cases, like `Region(0)`, `Region(-1)`, `Region(view.size() + 100)`, `0`, `-3` ...
additionally The ST API already states that they return a `str`
> substr(region) | str | Returns the contents of the region as a string.
> -- | -- | --
> substr(point) | str | Returns the character to the right of the point.